streams/src/main/java/org/apache/kafka/streams/kstream/KTable.java

```diff
@@ -696,40 +1590,45 @@
      * <td>&lt;K1:ValueJoiner(A,b)&gt;</td>
      * </tr>
      * <tr>
-     * <td>&lt;K1:C&gt;</td>
-     * <td>&lt;K1:C&gt;</td>
+     * <td>&lt;K1:null&gt;</td>
```

Comment:
why these changes?

Reply:
I think, these changes come from javadoc shift due to new methods added.